problems/5.loops.js

```diff
@@ -1,33 +1,48 @@
 /** Returns every even number from 0 to a given number
-* @param {number} num - The number to iterate up to
-* @param {number[]} - An array of even numbers from 0 to {num}
-*/
-function evenNumbers() {
+ * @param {number} num - The number to iterate up to
+ * @param {number[]} - An array of even numbers from 0 to {num}
+ */
+function evenNumbers(array) {
+    let newArray = [];
+    let arr = Math.floor(array.lenght % 2 === 0);
+    for (let i = 0; i < array.length; i++) {
```

Comment:
we want to iterate from 0 to num. No need for an array here.

Reply:
Ok. Thank you